katalog/etcd-backup/etcd-backup-s3/kustomization.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: kube-system
+
+resources:
+  - cronjob.yaml
+
+images:
+  - name: etcd
+    newName: registry.sighup.io/fury/etcd-backup/etcd
+    newTag: 3.5.17
+  - name: rclone
+    newName: registry.sighup.io/fury/etcd-backup/rclone
+    newTag: v1.69-stable
+
+patches:
+  - patch: |-
+      apiVersion: batch/v1
+      kind: CronJob
+      metadata:
+        name: etcd-backup-s3
+      spec:
+        schedule: "* * 1 * *"
```

Comment:
Why are we changing the schedule? just to test patches?

Reply:
Yes, my rationale behind this was to expose to a customer an *already made* patch to easily change the `schedule`.